assets/main.js

```diff
@@ -8,10 +8,12 @@ import {
 
 const main = document.querySelector('main');
 const pastLetters = [];
+const ATTEMPTS_LIMIT = 7
```

Comment:
Change to style guide project

```suggestion
const attemptsLimit = 7
```